File: c/PRAK505-2210817310005-IkhsanGymnastiar.c
#include<stdio.h> 
void biodata(int tahunLahir, char A[20], char B[15]){
     int tahun_sekarang = 2020;
     printf("Perkenalkan Nama Saya : %s\n", A);
     printf("Umur Saya : %d\n", tahun_sekarang - tahunLahir);
     printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
     printf("Asal Saya Dari : %s",B);}
int main (){
     int tahunLahir;
     char A[20], B[15];
     scanf(" %d", &tahunLahir);
     scanf(" %[^\n]%*c", &A);
     scanf(" %[^\n]%*c", &B);
     biodata(tahunLahir, A, B);     
     return 0;}

File: python/PRAK505-2210817310005-IkhsanGymnastiar.py
def Biodata(tahunLahir, Namaku, Asal):
    tahunsekarang = 2020
    print("perkenalkan Nama Saya : {} ".format(Namaku))      
    print("Umur Saya : {} ".format(tahunsekarang - tahunLahir))
    print("Asal Saya dari : {} ".format(Asal))
tahunLahir = int(input())
Namaku = input()
Asal = input()
Biodata(tahunLahir, Namaku, Asal)